ods_ci/tasks/Resources/Provisioning/GPU/AMD/blacklist_driver.yaml

```diff
@@ -10,7 +10,7 @@ spec:
       version: 3.2.0
     storage:
       files:
-        - path: "/etc/modprobe.d/amdgpu-blacklist.conf"
+        - path: "/etc/modprobe.d/blacklist-amdgpu.conf"
```

Comment:
Hi @bdattoma we should avoid naming files like black or white list.
Does this file come fromAMD operator is is this our own file?

Reply:
Folliwing instructions on AMD docs https://dcgpu.docs.amd.com/projects/gpu-operator/en/latest/drivers/installation.html